.circleci/config.yml

```diff
@@ -62,25 +71,42 @@ jobs:
           keys:
             - ios-yarn-cache-{{ checksum "yarn.lock" }}
 
-      - run: yarn install
+      - run:
+          name: Set Xcode Path
+          command: sudo xcode-select -s /Applications/Xcode.app
+
+      - run:
+          name: Install Node.js and Yarn Dependencies
+          command: yarn install
+
+      - run:
+          name: Build the Project
+          command: yarn run build
 
       - save_cache:
           key: ios-yarn-cache-{{ checksum "yarn.lock" }}
           paths:
             - ./node_modules
-            - ./apps/AEPSampleApp/node_modules
+            - ./apps/AEPSampleAppNewArchEnabled/node_modules
 
       - run:
-          name: Install latest Cocoapods
+          name: Install Cocoapods
           command: sudo gem install cocoapods
 
       - run:
-          name: Install pods
-          command: yarn sampleapp:ios:pod:install
+          name: Install Pods
+          command: yarn sampleappnewarchenabled:ios:pod:install
 
       - run:
           name: Building iOS Sample App
-          command: cd apps/AEPSampleApp && npx react-native run-ios --verbose --terminal terminal
+          command: |
+            cd apps/AEPSampleAppNewArchEnabled/ios
+            xcodebuild \
+              -workspace AEPSampleAppNewArchEnabled.xcworkspace \
+              -scheme AEPSampleAppNewArchEnabled \
+              -sdk iphonesimulator \
+              -destination "generic/platform=iOS Simulator" \
+              clean build
```

Comment:
Is Android app still having error?

Reply:
yes I have tried to run it on android but still facing the same

Reply:
Could you update the sample app to AEPSampleAppNewArchEnabled in the comments in the Android at the bottom of the file?

Reply:
sure

Reply:
> [CXX1210] /root/project/apps/**AEPSampleApp**/node_modules/react-native-reanimated/android/CMakeLists.txt debug|arm64-v8a : No compatible library found